integration/test/Test/Login.hs

```diff
@@ -0,0 +1,115 @@
+{-# OPTIONS_GHC -Wno-ambiguous-fields #-}
+
+module Test.Login where
+
+import API.BrigInternal (getVerificationCode)
+import API.Common (defPassword)
+import API.GalleyInternal
+import API.Nginz (login, loginWith2ndFactor)
+import Control.Concurrent (threadDelay)
+import qualified Data.Aeson as Aeson
+import SetupHelpers
+import Testlib.Prelude
+
+testLoginVerify6DigitEmailCodeSuccess :: HasCallStack => App ()
+testLoginVerify6DigitEmailCodeSuccess = do
+  (owner, team, []) <- createTeam OwnDomain 0
+  email <- owner %. "email"
+  setTeamFeatureLockStatus owner team "sndFactorPasswordChallenge" "unlocked"
+  setTeamFeatureStatus owner team "sndFactorPasswordChallenge" "enabled"
+  generateVerificationCode owner email
+  code <- getVerificationCode owner "login" >>= getJSON 200 >>= asString
+  bindResponse (loginWith2ndFactor owner email defPassword code) $ \resp -> do
+    resp.status `shouldMatchInt` 200
```

Comment:
in the old test, we were checking the json content as well, i think that was useful.

Reply:
Can't find it, can you point me to it?

Reply:
https://github.com/wireapp/wire-server/pull/3986/files#diff-5d67659df46f564af0bf4bf2cfa2977d5be77b859c66c70cb18fe7c6571b9393L449-L455

Reply:
sorry, i was reading it wrong.  all good!